src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/boss_salramm_the_fleshcrafter.cpp

```diff
@@ -57,7 +57,11 @@ class boss_salramm : public CreatureScript
         {
             boss_salrammAI(Creature* creature) : BossAI(creature, DATA_SALRAMM) { }
 
-            void InitializeAI() override { Talk(SAY_SPAWN); }
+            void InitializeAI() override {
```

Comment:
I believe he also meant that you kept the opening bracket { on the first line
instead of moving it down to its own line. Not sure if it is TC codestyle to do so,
but it seems to be the norm in the TC source (at least for consistency).